pkg/plugins/plugins.go

```diff
@@ -108,7 +116,9 @@ func (pm *PluginManager) startBackendPlugins(ctx context.Context) error {
 }
 
 func (pm *PluginManager) Run(ctx context.Context) error {
-	pm.startBackendPlugins(ctx)
+	if err := pm.startBackendPlugins(ctx); err != nil {
```

Comment:
Seems like `startBackendPlugins` cannot return an error so remove it as return argument?

Reply:
@marefr Might it be an idea to modify `startBackendPlugins` to return errors from `ds.startBackendPlugin`?

Reply:
I see no reason to return an error from `startBackendPlugins` since failing to start a backend plugin shouldn't fail to start Grafana. Maybe I'm missing something?